app/components/layout/Column.tsx

```diff
@@ -1,14 +1,38 @@
 import type { ReactNodeLike } from "prop-types";
+import { useEffect, useRef } from "react";
+import { gsap } from "gsap";
+import { ScrollTrigger } from "gsap/dist/ScrollTrigger";
```

Comment:
nit: Any reason these are two imports versus one? Does gsap not export ScrollTrigger from the base?